Darkly/LDClientManager.m

```diff
@@ -26,6 +27,7 @@ +(LDClientManager *)sharedInstance {
         
         [[NSNotificationCenter defaultCenter] addObserver:sharedApiManager selector:@selector(willEnterForeground) name:UIApplicationWillEnterForegroundNotification object:nil];
         [[NSNotificationCenter defaultCenter] addObserver:sharedApiManager selector:@selector(willEnterBackground) name:UIApplicationWillResignActiveNotification object:nil];
+        [[NSNotificationCenter defaultCenter] addObserver:sharedApiManager selector:@selector(syncWithServerForEvents) name:kLDBackgroundFetchInitiated object:nil];
```

Comment:
@jkodumal Developers would need to add the `application:performFetchWithCompletionHandler:` to their AppDelegate and post this notification from there to make the background fetching work. Or if you have a more efficient idea? Because this method does not throw it's own notification.